v1.1-beta/matrixx-digital-commerce/matrixx-tmf-usage-consumption-management/cnf-testsuite.yml

```diff
@@ -0,0 +1,12 @@
+---
+helm_directory: "tmf-usage-consumption-management"
+release_name: "cnf"
+helm_install_namespace: "matrixx"
+helm_values: "--values /home/dwilmes/.mtx/konstruxx/working/values486272589.yaml --devel"
```

Comment:
@deerowbear @daniel-wilmes generic please for end-users to be able to repeat testing if desired